Gjennomgang og nyskriving (#99)
DHP-stottepakke/pages: pages/dmp_supporting_info/support_09_responsibilities_resources.md

{% include callout.html type="tip" content="
**Why is this topic important**\\
>> Defining responsibilities, project involvement and cost related to research data management is highly important when planning and conducting research projects\\
>> Relevant estimates of costs for research data management should be included when applying for financing, and this chapter will help you to develop relevant estimates\\
" %}

## About this chapter

> In this chapter, you are asked to define responsibilities, who will be involded in the project, and the cost related to research data management. Remeber that additional expertice can be needed when covering the areas of responsibility in the project. Also remeber that cost can be relevant both for facilities, equipment, hardware, software, license costs or other recurring fees, preservation and archiving, costs for preparing data for publication and more. The questions in this chapter cover such aspects of research data management.       

Reviewer: ```suggestion
In this chapter, you are asked to define responsibilities, who will be involved in the project, and the cost related to research data management. Remember that additional expertise can be needed when covering the areas of responsibility in the project. Also remember that cost can be relevant both for facilities, equipment, hardware, software, license costs or other recurring fees, preservation and archiving, costs for preparing data for publication and more. The questions in this chapter cover such aspects of research data management.       
```

## Question-specific guidance

### Have data management responsibilities in the project been assigned?

> Please specify. 

Reviewer: ```suggestion
Being specific about data management responsibilities in a project is recommended. Contributors and their roles should be specified in the chapter 'About the Data Management Plan'. In educational settings, the supervisor(s) should be specified as they have formal responsibility for data management.
```

Reviewer: We should consider if this one should be changed to an 'item select' question.

Author: Yes, agree

### To execute the project, is additional expertise required?
In the context of research data management this could include:

> Data Stewards

Reviewer: ```suggestion
* Data Stewards
```

> Data Engineers

Reviewer: ```suggestion
* Data Engineers
```

> Software developers

Reviewer: ```suggestion
* Research software engineers
* Software developers
```

> Data entry specialists

Reviewer: ```suggestion
* Data entry specialists
```

> Data protection specialists

Reviewer: ```suggestion
* Data protection specialists
```

> Legal advisers

Reviewer: ```suggestion
* Legal advisors
```

### To execute the project, are arrangements needed to access facilities or equipment?
Necessary arrangements to access facilities or equipment may include training project members, arranging facility access, or executing security clearance. Consider that these processes can be time-consuming, and therfore should be planned for. 

### Will you require hardware or software in addition to what is usually available in the institute?
Please specify.

> ### Will there be (software) license costs or other recurring fees to keep data or documents available?

Reviewer: ```suggestion
### Will there be software license costs or other recurring fees to work with data or documents?
```

Reviewer: Comment to the KM to avoid 'availability' misunderstandings. Still looking for a better phrasing.

Please specify whether you are using any commercially licensed products to keep data, software or documents available, for which a regular fee must be paid.

### Will data storage in the active phase of the project be associated with costs?
Please specify whether data storage will come with a cost in the active phase. 

### Will data preservation and archiving be associated with costs?

> Please specify whether data preservation and archiving will be associated with costs towards the end of the project. 

Reviewer: ```suggestion
Please specify whether data preservation and archiving beyond the project phase will be associated with costs. This may include fees for data curation.
```

### Did you budget for the time and effort it will take to prepare the data for publication?

> Please specify whether preparation of data for publication will be associated with costs towards the end of the project. 

Reviewer: ```suggestion
Please specify whether preparation of data for publication will be associated with costs. Consider the time and effort required.
```
